Library/OcHdaDevicesLib/OcHdaDevicesLib.c

```diff
@@ -242,6 +242,8 @@ STATIC HDA_CODEC_LIST_ENTRY mHdaCodecList[] = {
   { HDA_CODEC (CONEXANT, 0x50B8),       0x0000, "Conexant CX20664" },
   { HDA_CODEC (CONEXANT, 0x50B9),       0x0000, "Conexant CX20665" },
   { HDA_CODEC (CONEXANT, 0xFFFF),       0x0000, "Conexant (Unknown)" },
+  { HDA_CODEC (CONEXANT, 0x50F4), 0x0000, "Conexant CX20724" },
+  { HDA_CODEC (CONEXANT, 0x2008), 0x0000, "Conexant CX8200" },
```

Comment:
Your IDs for these devices match similar code in Linux audio drivers, so thank you for providing them.

Do please remember keep the code layout (tabs, spacing) the same as code you are adding to - also notice that existing entries are ordered by increasing id, so please match that (your two entries need to go at two different places in the existing list).

If you could make those two changes it should be good to merge. Thank you.

Reply:
https://github.com/acidanthera/OpenCorePkg/pull/326/commits/4a4ba775afd27ce5701c29c08b591333b9adcf97